atlasdb-config/src/test/java/com/palantir/atlasdb/factory/AutoServiceAnnotatedAtlasDbFactory.java

```diff
@@ -44,7 +44,8 @@ public String getType() {
     }
 
     @Override
-    public KeyValueService createRawKeyValueService(KeyValueServiceConfig config, Optional<LeaderConfig> leaderConfig) {
+    public KeyValueService createRawKeyValueService(KeyValueServiceConfig config, Optional<LeaderConfig> leaderConfig,
```

Comment:
Why does this need `initializeAsync`?

Reply:
It implements the signature from `AtlasDbFactory`. The factory needs it to propagate the `initializeAsync` config down when creating the CassandraKVS.